### ISSUE TITLE
Problem as Janitor Borg

### ISSUE THREAD

**Issue by ColdDead** [space-wizards/space-station-14#24293]
## Description

Simply put, the Borg Janitor with its adv module cannot refill its ADV janitor module without a space cleaner refill station, which is a problem because not every map has one.

**Reproduction**

Just spray it empty and try to refill it on any chem container

**Screenshots**
![20240119145917_1](https://github.com/space-wizards/space-station-14/assets/38980900/eb0b10c7-4f42-4897-90d4-3a581a726be6)

**Additional context**

My idea is to simply insert a dropper and a 50u beaker into the ADV module like the medic borg already has in the ADV module

**Comment by Errant-4** [comment]
Wouldn't that allow the janiborg to easily load any available solution they choose into the sprayer?

**Comment by deltanedas** [comment]
good

**Comment by ColdDead** [comment]
> Wouldn't that allow the janiborg to easily load any available solution they choose into the sprayer?

Yes, but some way this need a way that the borg can do all stuff

**Comment by deltanedas** [comment]
firefighter borg and fire fighter borg

**Comment by UbaserB** [comment]
Simply map the space cleaner into all maps

**Comment by Errant-4** [comment]
Okay I tried (implementing this is trivially easy) and it's _annoying_ to fill the beaker with a dropper. I suppose it's better than nothing, but... Perhaps not an ideal solution. Simply making sure dispensers are present on all maps at janitorial would possibly be a bigger QoL improvement although a case could easily be made for going for both solutions simultaneously

**Comment by deltanedas** [comment]
syringe is faster (still annoying but better than dropper) but has the issue of janiborg validhunters having syringe AND spray

**Comment by Errant-4** [comment]
What implications would it have if their dropper was simply larger? Like 50u so it only needs 1 click. Would they be press-ganged by the crew to do chemistry faster, or could that be abused in some way?

**Comment by deltanedas** [comment]
beakers already transfer 50u so doesnt matter

**Comment by Errant-4** [comment]
> beakers already transfer 50u so doesnt matter

Yes but it takes 2x10 clicks to fill a beaker _once_

**Comment by deltanedas** [comment]
right click set transfer amount :trollface:

**Comment by Errant-4** [comment]
> right click set transfer amount :trollface:

Yes and the dropper by default has a maximum volume of 5 units dont trollface me if you haven't tried to do it
(Though I just saw the dropper on the adv medical module is already extended to 10u capacity

**Comment by deltanedas** [comment]
set transfer amount of beaker to 50u i mean, its no better to force a borg to transfer shit

**Comment by Errant-4** [comment]
> set transfer amount of beaker to 50u i mean, its no better to force a borg to transfer shit

Oh right. I forgot regular people don't NEED to use a beaker as an intermediary since they can just _put it down_. Okay then. So give the janiborg a 50u dropper and 50u beaker?

**Comment by perryprog** [comment]
Fixed by #38668, which gave advanced janiborg a droppable beaker slot.
